### PR TITLE
Update Documentation for provisioning new items.

### PR DESCRIPTION
**Pull request by oikeme**
Updating the documentation to include details for provisioning new stand alone items. Also made some changes to move all endpoint documentation in to individual files in the jetpack-start-endpoints folder.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,17 +9,17 @@ If you have any technical questions or concerns, don’t hesitate to get in touc
 - Jetpack
   - Plan provisioning
     - [Via CLI](jetpack/plan-provisioning.md)
-    - [Via direct API call](jetpack/plan-provisioning-direct-api.md)
+    - [Via direct API call](jetpack/jetpack-start-endpoints/provisioning.md)
```
Comment (ebinnion) [inline README.md]: This link is incorrect. The file is now called `plan-provisioning.md`.

File: README.md [hunk 2]
```diff
@@ -9,17 +9,17 @@ If you have any technical questions or concerns, don’t hesitate to get in touc
...
   - Checking status
     - [Connection status](jetpack/determining-connection-status.md)
     - [Provisioning status](jetpack/determining-provisioning-status.md)
   - [Managing modules](jetpack/managing-modules.md)
   - [Upgrade redirection](jetpack/upgrade-redirection.md)
   - [Monitor downtime notificatiosn webhook](jetpack/monitor-downtime-notifications-webhook.md)
-  - [Magic mobile links](jetpack/mobile-magic-link.md)
-  - [Setting Backups SSH credentials](jetpack/setting-backups-ssh-credentials.md)
-  - [Errors](jetpack/errors.md)
+  - [Magic mobile links](jetpack/jetpack-start/mobile-magic-link.md)
```
Comment (ebinnion) [inline README.md]: All of the `jetpack-start` paths should be `jetpack-start-endpoints` here.

File: jetpack/jetpack-start-endpoints/determining-provisioning-status.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+- Does a given site have a plan provisioned?
+- Was the plan provisioned by a given partner?
+- Is the site connected to WordPress.com?
+
+To provide the provisioning status of a site, there is the `/jpphp/{$site}/status` endpoint.
+
+## Getting an access token
+
+To query this status endpoint, you'll first need to retrieve an access token. Information about that can be found on the [plan provisioning via API document](../plan-provisioning.md#getting-a-jetpack-partner-access-token ).
```
Comment (ebinnion) [inline jetpack/jetpack-start-endpoints/determining-provisioning-status.md]: This should probably link to the new authentication document.

File: jetpack/jetpack-start-endpoints/determining-provisioning-status.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+- __pretty__:        Defaults to `false`. Setting to `true` will output pretty JSON.
+
+## Response Parameters
+
+- __plan__:                              (string) The slug of the provisioned plan or empty string when no plan.
+- __is_partner_plan__:                   (bool) Does the site have a plan that was provisioned by a partner?
+- __is_provisioned_by_calling_partner__: (bool) Does the site have a plan that was provisioned by the partner making this status request?
+- __is_connected__:                      (bool) Is the site connected to WordPress.com?
+- __has_pending_plan__:                  (bool) Does the site have a pending plan that requires connection authorization?
+- __products__: Additional product items provisioned for the site including jetpack backups products and other items.
```
Comment (ebinnion) [inline jetpack/jetpack-start-endpoints/determining-provisioning-status.md]: Let's capitalize "jetpack" here.

File: jetpack/jetpack-start-endpoints/plan-cancellation.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+`$site` is the site's domain and path where `/` in the path is replaced with `::`. For example:
+
+| Site URL              | $site Identifier        |
+| --------------------- | -------------------     |
+| `example.com`         | `example.com`           |
+| `example.com/demo`    | `example.com::demo`     |
+| `example.com/demo/wp` | `example.com::demo::wp` |
+
+### Request Parameters (/partner-cancel)
+- __plan__: A slug representing which plan or item to cancel. One of `free`, `personal`, `premium`, or `professional` for plans. For backups either `jetpack-backup-realtime` or `jetpack-baackup-daily`. Note this is an optional parameter for Jetpack Plans (Personal, Premium, Professional) but required for cancelling standalone items including Jetpack backups.
```
Comment (ebinnion) [inline jetpack/jetpack-start-endpoints/plan-cancellation.md]: `jetpack-baackup-daily` is a typo.

File: jetpack/jetpack-start-endpoints/plan-cancellation.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+- __cancel-all__: Optional. If this option is set to true ALL purchased items provisioned by the calling partner for a given site URL will be cancelled.
```
Comment (ebinnion) [inline jetpack/jetpack-start-endpoints/plan-cancellation.md]: I'd probably add a comma after `true`.

File: jetpack/jetpack-start-endpoints/plan-cancellation.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+### Query Parameters (/partner-cancel)
+- __http_envelope__: Default to `false`. Sending `true` will force the HTTP status code to always be `200`. The JSON response is wrapped in an envelope containing the "real" HTTP status code and headers.
+- __pretty__:        Defaults to `false`. Setting to `true` will output pretty JSON.
+
+### Response Parameters (/partner-cancel)
+
+Below, the response parameters are grouped by whether the request to cancel errored or not.
+
+#### Successful response (/partner-cancel)
```
Comment (ebinnion) [inline jetpack/jetpack-start-endpoints/plan-cancellation.md:31]: This section needs to be updated for the `cancel-all` case where we add some extra information to the response.

File: jetpack/jetpack-start-endpoints/plan-provisioning.md [hunk 8]
```diff
@@ -1,94 +1,9 @@
-# Provisioning and Cancelling Jetpack Plans – Querying the API Directly
+# Provisioning and Cancelling Jetpack Plans
 
 In [another document](plan-provisioning.md), we discussed how to provision and cancel plans by using the shell script that ships with Jetpack. But, for partners where running shell commands won't work, it is possible to communicate directly to the API on WordPress.com.
 
 If you have any questions or issues, our contact information can be found on the [README.md document](../README.md).
```
Comment (ebinnion) [inline jetpack/jetpack-start-endpoints/plan-provisioning.md]: This link is broken now.

File: jetpack/jetpack-start-endpoints/plan-provisioning.md [hunk 9]
```diff
@@ -1,94 +1,9 @@
-# Provisioning and Cancelling Jetpack Plans – Querying the API Directly
+# Provisioning and Cancelling Jetpack Plans
 
 In [another document](plan-provisioning.md), we discussed how to provision and cancel plans by using the shell script that ships with Jetpack. But, for partners where running shell commands won't work, it is possible to communicate directly to the API on WordPress.com.
```
Comment (ebinnion) [inline jetpack/jetpack-start-endpoints/plan-provisioning.md]: This link is incorrect now. It should point to the shell provisioning doc.

File: jetpack/jetpack-start-endpoints/plan-provisioning.md [hunk 10]
```diff
@@ -102,7 +17,7 @@ Plans can be provisioned by making a request using your partner token from the s
 
 - __local_user__:      The username, ID or email on the local website (not the WordPress.com username) that should own the plan. The corresponding user _must_ already exist.
 - __siteurl__:         The URL where the WordPress core files reside.
-- __plan__:            A slug representing which plan to provision. One of `free`, `personal`, `premium`, or `professional`.
+- __plan__:            A slug representing which plan or item to provision. One of `free`, `personal`, `premium`, or `professional` for plans. For backups either `jetpack-backup-realtime` or `jetpack-baackup-daily`
```
Comment (ebinnion) [inline jetpack/jetpack-start-endpoints/plan-provisioning.md]: There is a typo in `jetpack-baackup-daily`.

Also, I'd probably add a comma after "For backups".

File: jetpack/jetpack-start-endpoints/provisioning-status.md [hunk 11]
```diff
@@ -10,7 +10,7 @@ To provide the provisioning status of a site, there is the `/jpphp/{$site}/statu
 
 ## Getting an access token
 
-To query this status endpoint, you'll first need to retrieve an access token. Information about that can be found on the [plan provisioning via API document]( plan-provisioning-direct-api.md#getting-a-jetpack-partner-access-token ).
+To query this status endpoint, you'll first need to retrieve an access token. Information about that can be found on the [plan provisioning via API document](../plan-provisioning.md#getting-a-jetpack-partner-access-token ).
```
Comment (ebinnion) [inline jetpack/jetpack-start-endpoints/provisioning-status.md]: This should probably link to the new authentication doc.

File: jetpack/jetpack-start-endpoints/setting-backups-ssh-credentials.md [hunk 12]
```diff
@@ -4,10 +4,9 @@ There are two ways to set Jetpack Backups SSH credentials. One way of doing this
 
 ## Getting an access token
 
-To query this status endpoint, you'll first need to retrieve an access token. Information about that can be found on the [plan provisioning via API document]( plan-provisioning-direct-api.md#getting-a-jetpack-partner-access-token ).
+To query this status endpoint, you'll first need to retrieve an access token. Information about that can be found on the [plan provisioning via API document]( provisioning.md#getting-a-jetpack-partner-access-token ).
```
Comment (ebinnion) [inline jetpack/jetpack-start-endpoints/setting-backups-ssh-credentials.md]: This should probably link to the new authentication doc.

File: users/user-creation.md [hunk 13]
```diff
@@ -8,7 +8,7 @@ To successfully create a WordPress.com account for a given user, there are two p
 - There must not be an existing WordPress.com account for the given email.
 
 ## Authentication
-In order to make use of this endpoint you will need to be authenticated. Information on this can be found [here](/jetpack/plan-provisioning-direct-api.md###endpoint-information)
+In order to make use of this endpoint you will need to be authenticated. Information on this can be found [here](/jetpack/jetpack-start-endpoints/plan-provisioning.md###endpoint-information)
```
Comment (ebinnion) [inline users/user-creation.md:11]: This should probably link to the new authentication document.

**Review by ebinnion — CHANGES_REQUESTED** [review]
Overall, I like the approach to breaking the documentation out.

There a few typos in product slugs that we need to fix, some broken links, and some links that need to be updated. Other than that, this looks pretty good.

File: jetpack/jetpack-start-endpoints/plan-cancellation.md (hunk 7 above)
Comment (oikeme) [reply jetpack/jetpack-start-endpoints/plan-cancellation.md:31]: Added a section for cancel-all to the response documentation

Comment (ebinnion) [reply jetpack/jetpack-start-endpoints/plan-cancellation.md:31]: Considering that the response would look something like this when `cancel-all` is set in the request:

```
{
    "pending_activations": [],
    "activations": [],
    "success": true
}
```

This is what I would suggest:

>
> - __success__:       (bool) Was the operation successful?. It is possible for success to be false if a plan did not exist for the site. Note: If `cancel-all` was set in the request, then the value will be `true` only if ALL items for the site were successfully cancelled. 
>
> - __pending_activations__: (conditional) (array) If `cancel-all` if set for the request, then this property will be added to the response to indicated all pending activations that were cancelled.
>
> - __activations__: (conditional) (array) If `cancel-all` if set for the request, then this property will be added to the response to indicated all activations that were cancelled.

**Review by ebinnion — APPROVED** [review]

**Pull request merged**